### ISSUE TITLE
Run rt object classification on RPi (with camera)

### ISSUE THREAD

**Issue by MarsiaSec** [lbo462/starflit#49]
using tflite_runtime

**Comment by MarsiaSec** [comment]
using [this](https://github.com/raspberrypi/picamera2/tree/main/examples/tensorflow):

 - good performance with mobilenetv2 (inference is good but accuracy could be a lot better)
 - not so good with yolov5 but can probably improve

need more tests and try finetuning
